monkey/infection_monkey/network/smbfinger.py

```diff
@@ -120,27 +120,26 @@ def get_host_fingerprint(self, host):
             n = SMBNego(data=SMBNegoFingerData())
             n.calculate()
 
-            Packet = str(h) + str(n)
-            Buffer = struct.pack(">i", len(''.join(Packet))) + Packet
-            s.send(Buffer)
+            packet_ = str(h) + str(n)
```

Comment:
Why is the variable named `packet_` and not `packet`?

Reply:
Not to be confused with class, no big difference.